install.sh

```diff
@@ -11,6 +11,30 @@ then
   abort "Bash is required to interpret this script."
 fi
 
+is_true() {
+  [[ "$1" =~ (yes|Yes|YES|true|True|TRUE) ]]
+}
+
+is_false() {
+  [[ "$1" =~ (no|No|NO|false|False|FALSE) ]]
+}
+
+# Allow literal values for CI (yes, true, 1)
```

Comment:
Have we had any requests for this? I think it's probably fine to just assume a non-empty string is true here.

Reply:
> Have we had any requests for this?

If the only use case for variable `NONINTERACTIVE` is the CI test, we can remove this, because we know what we are doing (non-empty for true, empty for false).

If we expose this to the users, I think the values `true / yes` and `false / no` are more intuitional.

Reply:
> If the only use case for variable `NONINTERACTIVE` is the CI test, we can remove this, because we know what we are doing (non-empty for true, empty for false).

Yes, that's fine.

> If we expose this to the users, I think the values `true / yes` and `false / no` are more intuitional. For example in the `Dockerfile`:

I disagree, that's not how Homebrew itself works. I've never seen `FOO=false` used in Bash/CLI applications.

Reply:
> I disagree, that's not how Homebrew itself works. I've never seen FOO=false used in Bash/CLI applications.

Alright, I will remove this.